ui/component/subscribeButton/view.jsx

```diff
@@ -8,17 +8,10 @@ import { useIsMobile } from 'effects/use-screensize';
 import { ENABLE_UI_NOTIFICATIONS } from 'config';
 import useBrowserNotifications from '$web/component/browserNotificationSettings/use-browser-notifications';
 
-type SubscriptionArgs = {
-  channelName: string,
-  uri: string,
-  notificationsDisabled?: boolean,
-};
-
 type Props = {
   permanentUrl: ?string,
   isSubscribed: boolean,
-  doChannelSubscribe: (SubscriptionArgs, boolean) => void,
-  doChannelUnsubscribe: (SubscriptionArgs, boolean) => void,
+  doToggleSubscription: (Subscription, boolean) => void,
```

Comment:
1. Unfollow @DarkHorsePodcastClips:b
2. Go to http://localhost:9090/@thomas.zarebczan:c
3. Click follow on Dark Horse
    - <img width="100" alt="image" src="https://user-images.githubusercontent.com/64950861/153531348-5ac00cf6-f674-4b25-9e7e-b3bd66337627.png">
4. Click on the bell
    - <img width="100" alt="image" src="https://user-images.githubusercontent.com/64950861/153531423-4bd9f7b6-7ec1-480f-8d25-2cc78df75385.png">
    - On production, it works. In this branch, it unfollows instead, although the toast is displaying the right message.